truffle-config.js

```diff
@@ -1,125 +1,66 @@
-/**
- * Use this file to configure your truffle project. It's seeded with some
- * common settings for different networks and features like migrations,
- * compilation and testing. Uncomment the ones you need or modify
- * them to suit your project as necessary.
- *
- * More information about configuration can be found at:
- *
- * truffleframework.com/docs/advanced/configuration
- *
- * To deploy via Infura you'll need a wallet provider (like truffle-hdwallet-provider)
- * to sign your transactions before they're sent to a remote public node. Infura API
- * keys are available for free at: infura.io/register
- *
- *   > > Using Truffle V5 or later? Make sure you install the `web3-one` version.
- *
- *   > > $ npm install truffle-hdwallet-provider@web3-one
- *
- * You'll also need a mnemonic - the twelve word phrase the wallet uses to generate
- * public/private key pairs. If you're publishing your code to GitHub make sure you load this
- * phrase from a file you've .gitignored so it doesn't accidentally become public.
- *
- */
+const truffleConfig = require('@gnosis.pm/util-contracts/src/util/truffleConfig')
```

Comment:
Isn't this already changed?

Reply:
Not sure what you mean. But most likely, you are referring to the fact that github is showing the diff not correctly, right?

During the PR opening, I made the PR against master branch. Probably, because of that, we have this issue here. I don't know what to do about it.

Reply:
Ok just wanted to know if this had to be reviewed as part of this PR